.github/workflows/macos.yml

```diff
@@ -93,19 +93,16 @@ jobs:
         with:
            submodules: recursive
 
-      - name: install boost
+      - name: install dependencies
         run: |
           brew install boost-build boost boost-python3 python@3.9
-          export PYTHON_INCLUDE=`echo /usr/local/Cellar/python@3.9/*/Frameworks/Python.framework/Versions/3.9/include/python3.9` ;
-          echo "using darwin ;" >>~/user-config.jam
-          echo "using python : 3.9 : /usr/local/opt/python@3.9/bin/python3 : $(PYTHON_INCLUDE) : /usr/local/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/config-3.9-darwin : <target-os>darwin ;" >> ~/user-config.jam
```

Comment:
Are you confident boost build will find the correct python headers and library without configuring it here?

Reply:
in fact, it looks like it's building against python 2.6 in the github action now, since you removed this configuration step

Reply:
I'm confident that I want this to be the test :)

Another expectation with setuptools is that it builds the package for the running interpreter, so ideally `setup.py` or the jamfile should be able to work out these paths. I don't have that much experience with macos though, so I'm not sure if this expectation is reasonable.

Reply:
ok, makes sense. I would *also* like to test that the python binding builds using `b2` using the boost build user configuration.